pkgs/development/python-modules/jax/default.nix

```diff
@@ -27,17 +27,16 @@ let
 in
 buildPythonPackage rec {
   pname = "jax";
-  version = "0.4.5";
+  version = "0.4.8";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = pname;
-    # google/jax contains tags for jax and jaxlib. Only use jax tags!
-    rev = "refs/tags/${pname}-v${version}";
-    hash = "sha256-UJzX8zP3qaEUIV5hPJhiGiLJO7k8p962MHWxIHDY1ZA=";
+    rev = "jax-v${version}";
```

Comment:
please don't change `rev` and its associated comment. nixpkgs as a whole is trying to move towards specifying refs/tags/... for everything since there can be name collisions between branches and tags.

Reply:
Understood !